### ISSUE TITLE
Data for US Counties no longer being updated

### ISSUE THREAD

**Issue by awakefie** [CSSEGISandData/COVID-19#1207]
If we are no longer going to update at the county level can we just remove that from the dataset?

**Comment by muneebaahmad** [comment]
@awakefie I am curious to know if you have found any other source for data by county.

**Comment by jayrbolton** [comment]
@CSSEGISandData We badly need some consistency around this issue. 

Remove counties if they're not updated. Most importantly, commit to having a dataset where all entries that fall underneath a country sum to the correct totals for that whole country without any overlap.

Later on, if you end up adding a sub-province level of detail, then make a separate folder with separate CSV files that have additional columns for country (sub-province) and city. Leave the original files to just the country/province levels so that nothing depending on them breaks. All city-level entries should sum to correct county totals, all county-level entries should sum to correct state/province-level totals, and all state/province level totals should sum to correct country-level totals.

P.S. Thanks for this project.

**Comment by dhmacq** [comment]
No, the counties shouldn't be removed, because their data is still needed for dates prior to 10-March, for which state level data is not provided.

In other words, to get a complete time series for California (for example) one has to add the CA counties and cities for dates through 9-March, then follow that by the CA data for 10-March and subsequent. Unless and until the older CA state level data is back-filled, it would be very bad to remove the city/county level data.

**Comment by jayrbolton** [comment]
Okay, I vote they should remove the counties and back-fill the dates into the state entries for the existing files. Then create separate data sources that include the sub-province level in a fully consistent way, if that's ever a goal.

**Comment by baselsalam** [comment]
Yes please, don't remove county level data.
Or give us the code you used to collect County Level data.

I'm getting requests from Doctors and Medical directors across the nation to get county-level data for their regions, to better understand what they need to prepare for. 

This is critical because different counties have different levels of lockdown in place and are growing at different rates.

**Comment by tautme** [comment]
I just now realize that the counties in California do not sum to the California observation. I am working on a proximity calculation and it does not work without county level. Does anyone know of other datasets that are more transparent about the collection methods? Message me if you want to start a group focused on finding one.

**Comment by dhmacq** [comment]
https://github.com/tautme, Did you include the cities in California along with the counties?

Which California observation is it that you say does not sum?

Until the state level values have been backfilled, there can be no comparing with county/city level data using the time series data files. They've done the backfill calculation (see pull request # 1259) and that calculation agrees with mine. But I don't see backfilled Calif data yet in time_series_19-covid-Confirmed.csv.

**Comment by paulboal** [comment]
The dashboard right now seems to have US county-level data in it now, but I don't see that in the files.  Where is the county-level data coming from, please?
<img width="764" alt="county-level" src="https://user-images.githubusercontent.com/1817916/77284797-837e9e00-6c9d-11ea-9291-3347f6f03df3.png">

**Comment by tautme** [comment]
@dhmacq Yes, I summed counties and cities. Tell me if I am wrong, I get 26 cities and counties in CA? With latest update. San Diego County, CA has zero cases on March 22...?

**Comment by tautme** [comment]
@paulboal I don't think they are sharing the complete dataset anymore. For instance, Arkansas shows differentiated locations across the state, but the data does not. If you look at Reuters map, it is obvious that they have for other states, but not Arkansas. 
<img width="922" alt="Screen Shot 2020-03-23 at 9 08 28 AM" src="https://user-images.githubusercontent.com/15580867/77325242-f14eb800-6ce5-11ea-982b-44af3f02c051.png">

**Comment by drewmarticorenaAWHERE** [comment]
@tautme I agree, the counts on the map appear to be systematically higher.  I am wondering if its some kind of bug due to lack of sleep on their part at this point

**Comment by tautme** [comment]
Count Negative test. The Arkansas Health Department is reporting at county level.
https://adem.maps.arcgis.com/apps/opsdashboard/index.html#/f533ac8a8b6040e5896b05b47b17a647

Anyone know where we can get data on the number of test administered?

**Comment by awakefie** [comment]
A few things.  I am open to any option that gets us to consistent data.  I would be over joyed to have county or city level data consistently.  Either way we need columns to aid in parsing.  Right now if I sum by 'US' the set seems way off.  Ultimately, the best way to handle this would be to split out the files by countries, states, counties, cities, etc.  That way you can choose what to parse and what you don't want to parse.

**Comment by awakefie** [comment]
USAfacts.org seems to be doing some good work around county data.  https://usafacts.org/visualizations/coronavirus-covid-19-spread-map/

The only thing here is looking for a consistent update frequency.  I am going to try and load this a secondary dataset and see how it looks.  Also, I work for a large healthcare institution which has data management as one of its specialties.  I am trying to drive that division to work on providing a public dataset.  Will update here if I am successful.

**Comment by dhmacq** [comment]
@tautme, yes, I find 26 cities and counties for California in the time series file. The more recent zeros are because they stopped reporting county/city data on 10-March and switched to state reporting. This was announced (somewhere) and they more recently announced (#1250) a change that will bring back county level data.

**Comment by tautme** [comment]
@dhmacq Thanks for sharing. I guess I was relying too much on things not changing. I am working on calculating proximity count by personalized locations, if anyone is interested.

**Comment by muneebaahmad** [comment]
If anyone is still looking, I found this website with county level data. https://coronadatascraper.com/#timeseries-byLocation.json
